claat/proto-renderer/testing-utils/general.go

```diff
@@ -8,31 +8,32 @@ import (
 	"runtime"
 	"testing"
 
+	"github.com/googlecodelabs/tools/claat/proto-renderer"
 	"github.com/googlecodelabs/tools/third_party"
 )
 
 // unsupportedType is a dummy type used to showcase the failures of rendering
 // non-proto custom types since we take in "any" type as rendering input.
 type UnsupportedType struct{}
 
-// Simple proto constructor
+// NewDummyProto is a simple proto constructor
 func NewDummyProto(in string) *tutorial.StylizedText {
```

Comment:
If this only returns stylized text, should it be called "NewStylizedText"?

Reply:
I wanted abstract out what type this really is, for readability distincion. No knowledge of actual protos is assumed on the tests (output-format base code) where this is used.

I'll make it not take in an argument and simply have "dummy" inside.